python_scripts/cross_validation_baseline.py

```diff
@@ -1,23 +1,18 @@
 # %% [markdown]
 # # Comparing results with baseline and chance level
```

Comment:
The title needs to be changed "Comparing results with baseline" or something similar since we don't mention chance level any more.

Reply:
Good points. Thanks for the review, I will push some fixes.